### PR TITLE
Develop -> Main

### PR DESCRIPTION
**Pull request by ddecrulle**
It's been too long since the develop branch was merged into main. In the future, we need to avoid this and provide stabilized versions on a regular basis. 

As long as this PR has not been validated, we won't accept any PRs on develop containing new features. 

Then I'll review all the CIs. 

Finally, we may also consider deleting the develop branch, which will require frequent mergers with main.

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -12,13 +12,13 @@ import { defineConfig, devices } from '@playwright/test';
...
 	/* Maximum time one test can run for. */
-	timeout: 5 * 1000,
+	timeout: 30_000,
 	expect: {
 		/**
 		 * Maximum time expect() should wait for the condition to be met.
 		 * For example in `await expect(locator).toHaveText();`
 		 */
-		timeout: 5000,
+		timeout: 30_000,
```
Comment (ddecrulle) [inline playwright.config.ts:21]: 30_000 instead of 30000 ?

File: src/components/commons/components/combo-box/combo-box-container.tsx [hunk 2]
```diff
@@ -1,28 +1,44 @@
...
-			id={`lunatic-combo-box-container-${id}`}
+			id={`${classNamePrefix ?? 'lunatic'}-combo-box-container-${id}`}
 			className={classnames(
-				'lunatic-combo-box-container',
 				className,
+				`${classNamePrefix ?? 'lunatic'}-combo-box-container`,
+				`${classNamePrefix ?? 'lunatic'}-suggester-${classStyle}`,
+				`lunatic-suggester-default-style`,
 				classStyle
 			)}
```
Comment (ddecrulle) [inline src/components/commons/components/combo-box/combo-box-container.tsx:32]: This result in a classname change even when classNamePrefix is undefined and could explain style issue

File: src/components/commons/components/combo-box/combo-box.stories.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import ComboBox from './combo-box';
 import { Meta, Story } from '@storybook/react';
-import { ComboBoxOption } from './combo-box.type';
+import { ComboBoxOptionType } from './combo-box.type';
 import { ComponentProps } from 'react';
```
Comment (ddecrulle) [inline src/components/commons/components/combo-box/combo-box.stories.tsx:4]: It's type so should be imported as type

File: src/components/commons/components/combo-box/combo-box-content-box.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import React, { PropsWithChildren } from 'react';
```
Comment (ddecrulle) [inline src/components/commons/components/combo-box/combo-box-content-box.tsx:1]: {type PropsWithChildren}

File: src/components/commons/components/combo-box/combo-box.tsx [hunk 5]
```diff
@@ -1,37 +1,38 @@
 import { useCallback, useReducer, useEffect, ReactNode } from 'react';
```
Comment (ddecrulle) [inline src/components/commons/components/combo-box/combo-box.tsx:1]: , type ReactNode

File: src/components/commons/components/combo-box/combo-box.tsx [hunk 6]
```diff
@@ -1,37 +1,38 @@
...
 import './combo-box.scss';
-import { ComboBoxOption } from './combo-box.type';
+import { ComboBoxOptionType } from './combo-box.type';
 import { SelectionProps, Selection } from './selection/selection';
 import { Panel, PanelProps } from './panel/panel';
-import { ComboBoxContainer } from './combo-box-container';
+import ComboBoxContainer from './combo-box-container';
 import ComboBoxContent from './combo-box-content';
 import { LunaticBaseProps } from '../../../type';
 import Label from '../label';
```
Comment (ddecrulle) [inline src/components/commons/components/combo-box/combo-box.tsx:12]: type

File: src/components/commons/components/combo-box/combo-box.tsx [hunk 7]
```diff
@@ -1,37 +1,38 @@
 import { useCallback, useReducer, useEffect, ReactNode } from 'react';
 import classnames from 'classnames';
-import { Delete } from './selection/delete';
+import { ClearButton } from './selection/clear-button';
 import { INITIAL_STATE, reducer, actions } from './state-management';
 import './combo-box.scss';
-import { ComboBoxOption } from './combo-box.type';
+import { ComboBoxOptionType } from './combo-box.type';
 import { SelectionProps, Selection } from './selection/selection';
 import { Panel, PanelProps } from './panel/panel';
```
Comment (ddecrulle) [inline src/components/commons/components/combo-box/combo-box.tsx:8]: type

**Review by laurentC35 — APPROVED** [review]

File: src/components/commons/components/combo-box/selection/__snapshots__/selection.spec.tsx.snap [hunk 8]
```diff
@@ -13,9 +13,11 @@ exports[`Selection component > should render correctly when it is disabled 1`] =
   <div
     class="lunatic-combo-box-selected disabled"
   >
-    <div>
-      No selection
-    </div>
+    <span
+      class="placeholder"
+    >
+      hello!
```
Comment (Grafikart) [inline src/components/commons/components/combo-box/selection/__snapshots__/selection.spec.tsx.snap:19]: Ce snapshot m'étonne on ne devrait pas avoir un hello! non ?

**Review by Grafikart — APPROVED** [review]

**Pull request merged**